### PR TITLE
Update the kubernetes-event-source example to use Broker, rather than Channel.

### PR DESCRIPTION
**Pull request by Harwayne**
Part of #1021 

## Proposed Changes

- Switches the sample from using `Channel` to use `Broker`.

### CODE REVIEW

**Comment by Harwayne** [comment]
Ping.

File: docs/eventing/samples/kubernetes-event-source/README.md [hunk 1]
```diff
@@ -36,49 +33,44 @@ kubectl apply --filename serviceaccount.yaml
 ### Create Event Source for Kubernetes Events
 
 1. In order to receive events, you have to create a concrete Event Source for a
-   specific namespace. If you are wanting to consume events from a different
-   namespace or using a different `Service Account`, you need to modify the yaml
+   specific namespace. If you want to consume events from a different
+   namespace or use a different `Service Account`, you need to modify the yaml
```
Comment (samodell) [inline docs/eventing/samples/kubernetes-event-source/README.md]: Can we specify what yaml file the user would need to modify to do that?

File: docs/eventing/samples/kubernetes-event-source/README.md [hunk 2]
```diff
@@ -36,49 +33,44 @@ kubectl apply --filename serviceaccount.yaml
...
 
 ```shell
 kubectl apply --filename k8s-events.yaml
 ```
 
-### Subscriber
+### Trigger
 
 In order to check the `KubernetesEventSource` is fully working, we will create a
 simple Knative Service that dumps incoming messages to its log and create a
```
Comment (samodell) [inline docs/eventing/samples/kubernetes-event-source/README.md]:
```suggestion
simple Knative Service that dumps incoming messages to its log and creates a
```

File: docs/eventing/samples/kubernetes-event-source/README.md [hunk 3]
```diff
@@ -36,49 +33,44 @@ kubectl apply --filename serviceaccount.yaml
...
+container and immediately delete it.
 
 ```shell
+kubectl run busybox --image=busybox --restart=Never -- ls
 kubectl delete pod busybox
 ```
 
 ### Verify
 
 We will verify that the kubernetes events were sent into the Knative eventing
```
Comment (samodell) [inline docs/eventing/samples/kubernetes-event-source/README.md]:
```suggestion
We will verify that the Kubernetes events were sent into the Knative eventing
```

File: docs/eventing/samples/kubernetes-event-source/README.md (hunk 1 above)
Comment (Harwayne) [reply docs/eventing/samples/kubernetes-event-source/README.md]: Done.

**Pull request merged**